yt_dlp/extractor/common.py

```diff
@@ -247,6 +247,7 @@ class InfoExtractor:
                                  (For internal use only)
                                  * http_chunk_size Chunk size for HTTP downloads
                                  * ffmpeg_args     Extra arguments for ffmpeg downloader
+                    * role      Role value
```

Comment:
This needs a better description -- "Role value" means nothing -- at least explain that it applies to DASH formats and give possible values